keda/README.md

```diff
@@ -66,6 +66,7 @@ their default values.
 | `metricsServer.dnsPolicy`                                  | Defined the DNS policy for the metric server | `ClusterFirst`
 | `metricsServer.useHostNetwork`                             | Enable metric server to use host network  | `false`
 | `imagePullSecrets`                                         | Name of secret to use to pull images to use to pull Docker images | `[]` |
+| `customLabels`                                             | Custom labels to add into metadata | `{}` |
 | `additionalLabels`                                         | Additional labels to apply to KEDA workloads | `` |
```

Comment:
`additionalLabels`  doesn't work for you?

Reply:
@zroubalik additionalLabels is a string. If i could change it to a map ?

Reply:
@tomkerkhove ^^ is this being used anywhere else? I think that the change makes sense.

Reply:
It's out there but I don't know how many people are using it.

I think it's best to change that one to a map indeed.

Along with #177 we can ship a breaking Helm chart version since both are breaking.

WDYT @zroubalik ?

Reply:
LGTM